docs/tailwind.config.js

```diff
@@ -21,6 +21,14 @@ module.exports = {
           red: '#ED0000',
         },
       },
+      height: {
```

Comment:
Not sure what those are for, but assuming they are "magic numbers" used exactly *once* in the entire project, I would avoid adding these as "tokens" in the Tailwind theme. 

Instead, I'd be more explicit about the "yeah, this is one-off" nature of these, and use _arbitrary values_ directly in the markup instead:

`h-[31rem]`, `h-[38rem]`, `w-[42rem]`, `w-[85rem]` (or use the pixel values if you prefer:

`h-[124px]`, `h-[152px]`, `w-[168px]`, `w-[340px]`

This immediately brings attention to the fact this is "bespoke, one-off, break out of the design system" styles, and doesn't "pollute" the theme with values never used again.

If they are used often, please ignore all of this! 😄

Reply:
Ahh great advice, thank you! You're absolutely right, these are one-off. I'll move them into the `h-[31rem]` syntax.